docs/user/pull-requests.rst

```diff
@@ -24,11 +24,19 @@ Build status report
 
        GitHub build status reporting
 
-Warning banner
-    A warning banner is shown at the top of documentation pages
+Pull request banner
+    A pull request banner is shown at the top of documentation pages
     to let readers know that this version isn't the main version for the project.
 
-    .. note:: Warning banners are available only for :doc:`Sphinx projects </intro/getting-started-with-sphinx>`.
+    .. note:: Warning banners are available only for :doc:`Sphinx projects </intro/getting-started-with-sphinx>` or projects using :doc:`Read the Docs Addons </addons>`.
```

Comment:
```suggestion
    .. note:: Warning banners are available only for projects using :doc:`Read the Docs Addons </addons>`.
```

I think this is probably a better place for now -- and once we fully ship them, we can remove this.